db/migrate/20210309100557_create_users.rb

```diff
@@ -1,7 +1,7 @@
 class CreateUsers < ActiveRecord::Migration[6.1]
   def change
     create_table :users do |t|
-      t.string :email
+      t.string :email, null: false, default: ""
```

Comment:
@josephelias-rubyians  Hi, emails are supposed to be mandatory. If you provide blank string, it defeats the purpose of making it mandatory validations. Why do you need blank string as default?

Reply:
It won't create any issues. Will not saves the record for empty email. Saves only there are any strings rather than null.

![image](https://user-images.githubusercontent.com/79498012/110886202-4aa46280-830e-11eb-9b1a-0295cf2333e9.png)

Reply:
@josephelias-rubyians  Hi Joseph, There are two sides in this conversation.
1) What I was talking about. -> Coding practise
2) What you were talking about is logically maintaining the code to achieve the goal.

Logical maintenance can be faulty, thats why we have coding practises. Please follow the my earlier comment to avoid such defaulting strings.